contact.php

```diff
@@ -2,6 +2,63 @@
 /**
 * Template Name: Contact page
 */
+ 
+//response generation function
+$response = "";
+
+//function to generate response
+function my_contact_form_generate_response($type, $message){
+
+global $response;
+
+if($type == "success") $response = "<div class='success'><p>{$message}</p></div>";
+else $response = "<div class='error'><p>{$message}</p></div>";
+
+}
+
+//response messages
+$not_human       = "Human verification incorrect.";
+$missing_content = "Please supply all information.";
+$email_invalid   = "Email Address Invalid.";
+$message_unsent  = "Message was not sent. Try Again.";
+$message_sent    = "Thanks! Your message has been sent.";
+ 
+//user posted variables
+$name = $_POST['message_name'];
+$email = $_POST['message_email'];
+$message = $_POST['message_text'];
+$human = $_POST['message_human'];
+ 
+//php mailer variables
+$to = get_option('admin_email');
+$subject = "Someone sent a message from ".get_bloginfo('name');
+$headers = 'From: '. $email . "\r\n" .
+  'Reply-To: ' . $email . "\r\n";
+
+if(!$human == 0){
+  if($human != 2) my_contact_form_generate_response("error", $not_human); //not human!
+  else {
+ 
+    //validate email
+    if(!filter_var($email, FILTER_VALIDATE_EMAIL))
+      my_contact_form_generate_response("error", $email_invalid);
+    else //email is valid
+    {
+        //validate presence of name and message
+        if(empty($name) || empty($message)){
+          my_contact_form_generate_response("error", $missing_content);
+        }
+        else //ready to go!
+        {
+            $sent = wp_mail($to, $subject, strip_tags($message), $headers);
```

Comment:
`send_form_email.php` is the code that captures the form and sends it to the email